extensions/amp-stream-gallery/1.0/README.md

```diff
@@ -0,0 +1,438 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays multiple similar pieces of content at a time, for features like related products or articles.
+experimental: true
+bento: true
+---
+
+# Bento Stream Gallery
+
+## Usage
+
+The Bento Stream Gallery is for displaying multiple similar pieces of content at a time along a horizontal axis. To implement a more customized UX, see
+[`amp-base-carousel`](https://amp.dev/documentation/components/amp-base-carousel/).
+
+_Example:_
+
+In the following example, we use `bento-stream-gallery` to display its content, showing a varying amount of its children at a time according to the viewport size.
+
+```html
+<bento-stream-gallery>
+  <img src="img1.png">
+  <img src="img2.png">
+  <img src="img3.png">
+  <img src="img4.png">
+  <img src="img5.png">
+  <img src="img6.png">
+  <img src="img7.png">
+</bento-stream-gallery>
+```
```

Comment:
this can be removed since it's in the web component example

Reply:
removed